### DISCUSSION TITLE
Marker Style abhängig von Attributen

### DISCUSSION THREAD

**Discussion by david-kammh** [e-netze/datalinq-community#8] [Q&A]
Hallo!

gibt es eine einfache Möglichkeit, die Farbe der Marker in einer Karte abhängig von einem Attribut zu machen?
Es gäbe zwar die Möglichkeit map.AddMarker zu verwenden anstatt mit Geoelementen (@DLH.BeginGeoElementFor) zu arbeiten, jedoch fällt dann die Möglichkeit des Klicks auf die Tabelle mit Zoom auf dem entsprechenden Marker weg.

Danke für eure Hilfe!

**Comment by jugstalt** [comment]
Hallo,

richtig, wenn du Marker mit der Tabelle verknüpfen möchtest, funktioniert das nur mi ``@DLH.BeginGeoElementFor()``. Allerdings wird da der Standard Marker verwendet. 
Die Anforderung müsste erst implementiert werden. Um abzuschätzen, ob es möglich ist, würde ich dich noch um eine genauere Beschreibung deiner Anforderungen bitten:
* Was ist der Anwendungsfall
* Was sollte möglich sein
* ...

LG Jürgen

**Reply by david-kammh** [reply]
in diesem Fall geht es darum POIs, je nach Zuständigkeit, farblich unterschiedlich darzustellen. Ich denke aber auch an andere Karten, wo individuelle png's verwendet werden könnten, die am eigenen Server abgespeichert sind oder anderswo via einer URL (z.B. https://github.com/pointhi/leaflet-color-markers?tab=readme-ov-file)  zugegriffen wird. So lassen sich Karten dann doch schneller und einfacher lesen.

Also zusammengefasst: dass man in @DLH.BeginGeoElementFor eine individuelle iconUrl (wie bei map.addMarker) angeben kann

LG

**Reply by gis-eni** [reply]
wie in https://github.com/gis-eni/datalinq-forum/discussions/16 beschrieben

**Comment by ue-black** [comment]
Wann wird das ungefähr kommen? Wir hätten auch Bedarf :-)
